projects/1.20-fabric/assets/neepmeat/neepmeat/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+
+  "block.neepmeat.fluid_gauge": "流体存量计",
+  "block.neepmeat.fluid_gauge.lore_0": "根据储罐的填满程度发出红石信号。",
+  "block.neepmeat.item_gauge": "物品存量计",
+  "block.neepmeat.item_gauge.lore_0": "根据容器的填满程度发出红石信号。",
+
+  "block.neepmeat.converter": "流体充能器",
+  "block.neepmeat.transducer": "燃液换能器",
+  "block.neepmeat.power_flower_seeds": "能量花种子",
+  "block.neepmeat.power_flower_seeds.lore_0": "可种在泥土或草方块里。会生长成壮观的有机产能器。",
+  "block.neepmeat.power_flower_growth": "能量花增生体",
+  "block.neepmeat.power_flower_growth.lore_0": "能量花结构的基础。\n上方与空气接触的增生体会特化为光合器官，其余增生体会增加代谢率。\n施加骨粉或肉时会扩散。\n可用精准采集取下。",
+  "block.neepmeat.power_flower_fluid_port": "能量花流体口",
+  "block.neepmeat.power_flower_fluid_port.lore_0": "能将水和液态食物吸入能量花。",
+  "block.neepmeat.power_flower_controller": "能量花核心",
+  "block.neepmeat.power_flower_controller.lore_0": "能量花产出的缥缈燃液自此流出。\n需向能量花流体口送入水。\n可直接破坏取下。",
+  "block.neepmeat.fluid_exciter": "心脏机",
+  "block.neepmeat.fluid_exciter.lore_0": "下方接受缥缈燃液时，上部即向脉管网络供能。效率是标准马达的150%。",
+
+  "block.neepmeat.verrucous_stone": "疣斑石",
+  "block.neepmeat.verrucous_stone_port": "疣斑石口",
+
+  "block.neepmeat.item_buffer": "展示台",
+  "block.neepmeat.item_buffer.lore_0": "潜行时点击以调整容量。",
+  "screen.neepmeat.display_plate.capacity": "输入新容量",
+
+  "block.neepmeat.item_duct": "漏斗延长管",
+
+  "block.neepmeat.rusty_panel": "锈蚀金属横板",
+
+  "block.neepmeat.dirty_sink": "脏污洗手池",
+  "block.neepmeat.small_sink": "小型洗手池",
+  "block.neepmeat.dirty_toilet": "脏污马桶",
+  "block.neepmeat.television_off": "电视",
+  "block.neepmeat.television_static": "雪花屏电视",
+  "block.neepmeat.large_fan": "大型风扇",
+  "block.neepmeat.large_fan.lore_0": "3x3装饰性方块。",
+
+  "block.neepmeat.big_lever": "大型拉杆",
+
+  "block.neepmeat.router": "物品路由器",
+  "block.neepmeat.router.lore_0": "以简单物品过滤器分配物品。",
+  "block.neepmeat.advanced_router": "高级物品路由器",
+  "block.neepmeat.advanced_router.lore_0": "以多个高级过滤器分配物品。",
+  "block.neepmeat.grinder": "粉碎机",
+  "block.neepmeat.grinder.lore_0": "空手Shift右击以提取经验。",
+  "block.neepmeat.large_crusher": "大型粉碎机",
+  "block.neepmeat.alloy_kiln": "合金窑",
+  "block.neepmeat.collector": "脂肪收集器",
+  "block.neepmeat.crucible": "熬脂盆",
+  "block.neepmeat.casting_basin": "液压盆",
+  "block.neepmeat.hydraulic_press": "液压机",
+  "block.neepmeat.heater": "加热器",
+  "block.neepmeat.heater.lore_0": "加热面向的类熔炉方块。每提供60eJ/t，加工速度附加1倍原本速度。能令暴露于锈铜液的铜氧化。",
+  "block.neepmeat.trommel": "亵渎滚筒筛",
+  "block.neepmeat.integrator_egg": "融变生物机卵",
+  "block.neepmeat.integrator_egg.lore_0": "它在不停跳动，似在表达它对血液的渴望。",
+  "block.neepmeat.advanced_integrator_egg": "高级融变生物机卵",
+  "block.neepmeat.advanced_integrator": "高级融变生物机",
+  "block.neepmeat.advanced_integrator_structure": "高级融变生物机结构",
+
+  "block.neepmeat.deployer": "放置器",
+  "block.neepmeat.mixer": "混合机",
+  "block.neepmeat.mixer.lore_0": "从相邻储罐中取出流体与物品混合。",
+  "block.neepmeat.mixer.lore_1": "物品须通过管道或漏斗送入。",
+  "block.neepmeat.mixer.lore_2": "产物通过管道取出。",
+  "block.neepmeat.breaker": "机动破坏器",
+
+  "block.neepmeat.motor_unit": "马达",

```

```suggestion
  "block.neepmeat.motor_unit": "发动机",
```
我知道第一印象是汽车发动机，但它可指一切产生机械能的机器。而马达指电机。